artman/config/converter.py

```diff
@@ -170,10 +169,10 @@ def _calculate_git_repos_config(artifact_config, output_dir):
 
 
 def _calculate_proto_paths(proto_paths):
-  src_proto_path, excluded_proto_path = [], []
-  for proto_path in proto_paths:
-      if proto_path.startswith('-'):
-          excluded_proto_path.append(proto_path[1:])
-      else:
-          src_proto_path.append(proto_path)
-  return src_proto_path, excluded_proto_path
+    src_proto_path, excluded_proto_path = [], []
```

Comment:
I do not normally bring up linting issues in PRs, but can we make this be consistent four-space indenting, please?
I will add a linter later.

Reply:
I guess you made the comment before I pushed another commit. The linter did warn me, though sometime it is hard to track given we have already had many linter warnings.

Reply:
I filed an issue for adding a linter to CI last week. There are dozens of already existing violations. https://github.com/googleapis/artman/issues/277

Reply:
Will address that issue in a follow-up.